.github/workflows/build_wheels.yml

```
@@ -3,7 +3,8 @@ on:
   release:
     types: [ published ]
   workflow_dispatch:
-  pull_request:
+  pull_request: # Temporarily trigger on PRs for debugging, remove later
+    types: [opened, synchronize, reopened]
```

This already ran for every PR
```suggestion
  pull_request:
```